packages/connect/src/Detector.test.ts

```diff
@@ -35,9 +33,8 @@ describe('Initialize Detector without extension', () => {
   // DEACTIVATED for now due to chainSpecs - once Kusama specs are available this should be activated
```

Comment:
I'd prefer if you used `test.skip` then the syntax continues to be built by typescript.  Keep the DEACTIVATED comment though